.github/workflows/e2e-test.yml

```diff
@@ -10,22 +10,22 @@ on:
   workflow_dispatch:
 
 env:
-  E2E_GH_TOKEN:          ${{ secrets.E2E_GH_TOKEN }}
-  WCP_DEV_TOOLS_REPO:    ${{ secrets.WCP_DEV_TOOLS_REPO }}
-  WCP_DEV_TOOLS_BRANCH:  'trunk'
-  WCP_SERVER_REPO:       ${{ secrets.WCP_SERVER_REPO }}
-  WC_SUBSCRIPTIONS_REPO: ${{ secrets.WC_SUBSCRIPTIONS_REPO }}
-  E2E_BLOG_ID:           ${{ secrets.E2E_BLOG_ID }}
-  E2E_BLOG_TOKEN:        ${{ secrets.E2E_BLOG_TOKEN }}
-  E2E_USER_TOKEN:        ${{ secrets.E2E_USER_TOKEN }}
-  WC_E2E_SCREENSHOTS:    1
-  E2E_SLACK_CHANNEL:     ${{ secrets.E2E_SLACK_CHANNEL }}
-  E2E_SLACK_TOKEN:       ${{ secrets.E2E_SLACK_TOKEN }}
-  E2E_USE_LOCAL_SERVER:  false
-  E2E_RESULT_FILEPATH:   'tests/e2e/results.json'
-  WC_MIN_SUPPORTED_VERSION: '7.6.0'
-  NODE_ENV: 'test'
-  FORCE_E2E_DEPS_SETUP: true
+  E2E_GH_TOKEN:                  ${{ secrets.E2E_GH_TOKEN }}
+  WCP_DEV_TOOLS_REPO:            ${{ secrets.WCP_DEV_TOOLS_REPO }}
+  WCP_DEV_TOOLS_BRANCH:          'trunk'
+  TRANSACT_PLATFORM_SERVER_REPO: ${{ secrets.TRANSACT_PLATFORM_SERVER_REPO }}
```

Comment:
Without updating this secret, I think it's still working until we reuse the old server repo name. 

But anyway, we would still update the secret to use the new one now.

Reply:
@htdat I have already created a secret `TRANSACT_PLATFORM_SERVER_REPO` with the new repo url